pootle/apps/pootle_store/models.py

```diff
@@ -2033,11 +2038,15 @@ def get_max_unit_revision(self):
     def can_be_updated(self):
         return not self.obsolete
 
-    def get_parent(self):
+    def get_parents(self):
         if self.parent.is_translationproject():
-            return self.translation_project
+            parents = [self.translation_project]
         else:
-            return self.parent
+            parents = [self.parent]
+
+        parents.extend(self.parent_vf_treeitems.all())
```

Comment:
This is a global reply to all the equivalent comments.

We can check if `virtualfolder` app is in `INSTALLED_APPS` in every single place we query the DB for vfolders. I am going to add another commit to the PR in order to do that.